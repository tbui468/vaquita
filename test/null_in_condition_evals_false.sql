if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        moons int,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        moons, 
                        ring_system
                    ) 

values
        ("Venus", 10.0, 1, true),
        ("Earth", null, null, null),
        ("Mars", 0.0, 2, false);

select name from planets where mass > 5.0;
select name from planets where moons = 1;
select name from planets where ring_system = true;
select name from planets where mass < 5.0;
select name from planets where moons <> 1;
select name from planets where ring_system = false;
close sol;
drop database sol;
exit;
