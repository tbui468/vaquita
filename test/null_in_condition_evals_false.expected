created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
name
Venus
name
Venus
name
Venus
name
Mars
name
Mars
name
Mars
closed database sol
dropped database sol
